File: QE-Selenium-Java/Activity9.java
package demos;

import org.openqa.selenium.By;
import org.openqa.selenium.firefox.FirefoxDriver;
import org.openqa.selenium.interactions.Actions;

public class Activity9 {
    public static void main(String[] args) {
        FirefoxDriver driver = new FirefoxDriver();
        Actions builder = new Actions(driver);

        driver.get("https://training-support.net/webelements/keyboard-events");
        System.out.println(driver.getTitle());

        builder.sendKeys("Hello World").build().perform();

        String text = driver.findElement(By.xpath("//h1[contains(@class,'text-4xl')]")).getText();
        System.out.println("You typed:" + text);

        driver.quit();
    }
}

File: QE-Selenium-Python/Activity9.py
from selenium import webdriver
from selenium.webdriver.common.by import By
from selenium.webdriver.common.action_chains import ActionChains

driver = webdriver.Firefox()
builder = ActionChains(driver)

driver.get("https://training-support.net/webelements/keyboard-events")
print(driver.title)

builder.send_keys("Hello World").perform()

text = driver.find_element(By.XPATH, "//h1[contains(@class,'text-4xl')]").text
print("You typed:" + text)

driver.quit()